cookbook/Tip-to-use-statics-files-js-css.md

```diff
@@ -0,0 +1,35 @@
+# Static Files
+
+## How let the Yesod know about your file
+
+Yesod build the static files to use inside the project.
+
+Example: We put one file in project:
+
+	/static/css/my_css_file.css
+
+The project needs to be rebuilt to bring css_my_css_file_css under the scope. If you don't build, he won't know about the file and when you use css_my_css_file_css, it won't be in the scope.
+	
+So to every new file added in static files you should execute the follows commands.
+
+```
+stack clean  # You should skip this command if you only added a file to the project.
```

Comment:
Here a new notice.

Reply:
Does this work: https://github.com/yesodweb/yesod-cookbook/pull/45#discussion_r314682079 ?

Reply:
No... No..  [this comment](https://github.com/yesodweb/yesod-cookbook/pull/45#discussion_r314682079)

Reply:
Please, could you change this part to better way? I'm little new at haskell and yesod. So, you can do this better than I !!